File: 251229/007/007.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split('\n');
const [s_code, m_point, time] = input[0].split(' ');

class Spy {
    constructor(secretCode, meetingPoint, time) {
        this.secretCode = secretCode;
        this.meetingPoint = meetingPoint;
        this.time = time;
    }
}

const s = new Spy(s_code, m_point, time);

console.log(`secret code : ${s.secretCode}`);
console.log(`meeting point : ${s.meetingPoint}`);
console.log(`time : ${s.time}`);

File: 251229/007/007.py
code, point, time = input().split()
time = int(time)

class Secret:
    def __init__(self, code, point, time):
        self.code = code
        self.point = point
        self.time = time

secret = Secret(code, point, time)
print(f'secret code : {secret.code}')
print(f'meeting point : {secret.point}')
print(f'time : {secret.time}')
